pkg/handlers/webhandler.go

```diff
@@ -82,6 +87,11 @@ func (h *AppHandler) Handle(ctx context.Context, event sdk.Event) error {
 			h.SetStatus(err.Error(), o)
 			return err
 		}
+
+		// append route onto runtime objs list
+		route := h.CreateRoute(o)
+		runtimeObjs = append(runtimeObjs, route)
```

Comment:
Should we be backwards compatible if routing subdomain isn't set in the CR? I'm thinking about 1.x here.

Reply:
The updated PR appears to be backwards compatible, so I'll mark this as resolved.

* New instances have the new format
* Upgraded instances keep their existing URLs